supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -126,32 +1119,202 @@
+
+*Use it*: yes
+
+*Incorrect forms*: Any form omitting the spaces, hyphen or correct capitalization
+
+*See also*: xref:undertow[undertow], xref:undertow-heading[Undertow]
+
 [discrete]
 [[web-services]]
 ==== Web services (noun)
-*Description*: Use as two words.
+*Description*: Use "Web services" when referring to the general concept of Web services. Write as two words. Capitalize "Web" and write "services" in lower case.
 
 *Use it*: yes
 
-*Incorrect forms*: webservices
+*Incorrect forms*: webservices, web services, Web Services
 
 *See also*:
 
+[discrete]
+[[webservices-heading]]
+==== Web Services (noun)
+*Description*: Use "Web Services" when referring to the xref:webservices[webservices] subsystem in titles and headings. Write as two words. Capitalize both words. See xref:webservices[webservices] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Web services, web services, webservices
+
+*See also*: xref:webservices[webservices]
+
+[discrete]
+[[webservices]]
+==== webservices (noun)
+*Description*: The "webservices" subsystem is used to configure the Web services provider. Write in lower case as one word. See xref:webservices-heading[Web Services] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: web services, Web services, Web Services
+
+*See also*: xref:webservices-heading[Web Services]
+
+[discrete]
+[[weld]]
+==== weld (noun)
+*Description*: The "weld" subsystem is used to configure Contexts and Dependency Injection (CDI) functionality for JBoss EAP. Write in lower case. See xref:weld-heading[Weld] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Weld
+
+*See also*: xref:weld-heading[Weld]
+
+[discrete]
+[[weld-heading]]
+==== Weld (noun)
+*Description*: Use "Weld" when referring to the xref:weld[weld] subsystem in titles and headings. Capitalize the word. See xref:weld[weld] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: weld
+
+*See also*: xref:weld[weld]
+
 [discrete]
 [[windows-server]]
 ==== Windows Server (noun)
-*Description*: This uppercase term is correct when referring to Microsoft’s Windows Server product or Windows-specific commands and scripts like `standalone.bat`. "Microsoft" does not precede the product name.
+*Description*: Use "Windows Server" to refer to the Microsoft Windows Server product or Windows-specific commands and scripts such as `standalone.bat`. Do not precede the product name with "Microsoft".
+
+*Use it*: yes
+
+*Incorrect forms*: Microsoft Windows Server, Microsoft Windows, Windows
+
+*See also*: xref:microsoft-windows[Microsoft Windows]
+
+// ***********************
+// Terms starting with 'X'
+// ***********************
+
+[discrete]
+[[xml]]
+==== XML (noun)
```

Comment:
This entry is of general interest beyond this product. Move to the main glossary.

Reply:
I've removed this entry and will add it to the main glossary as part of a separate pull request in due course.